File: web/src/main/java/no.nav.sbl/db/domain/PEvent.java
package no.nav.sbl.db.domain;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class PEvent {
    public Long id;
    public String veilederIdent;
    public String eventType;
    public LocalDateTime created;
    public String verdi;
}

File: src/main/java/no/nav/sbl/db/domain/PEvent.kt
package no.nav.sbl.db.domain

import java.time.LocalDateTime

data class PEvent(
    var id: Long? = null,
    var veilederIdent: String? = null,
    var eventType: String? = null,
    var created: LocalDateTime? = null,
    var verdi: String? = null,
)
